src/main/webapp/i18n/de/userSettings.json

```diff
@@ -0,0 +1,46 @@
+{
+    "artemisApp": {
+        "userSettings": {
+            "settings": "Benachrichtigungseinstellungen",
+            "saveChanges": "Änderungen speichern",
+            "saveSettingsSuccessAlert": "Die Einstellungen wurden erfolgreich gespeichert.",
+            "userSettings": "Benutzereinstellungen",
+            "accountInformation": "Account Informationen",
+            "notificationSettings": "Benachrichtigungen",
+            "accountInformationPage": {
+                "fullName": "Vollständiger Name",
+                "login": "Login",
+                "email": "Email",
+                "joinedArtemis": "Artemis beigetreten am"
+            },
+            "categories": {
+                "NOTIFICATION_SETTINGS": "Benachrichtigungseinstellungen"
+            },
+            "optionGroupNames": {
+                "Exercise Notifications": "Aufgabenbenachrichtigungen",
+                "Lecture Notifications": "Vorlesungsbenachrichtigungen",
+                "Instructor Exclusive Notifications": "Exklusive Benachrichtigungen für Lehrende"
```

Comment:
why not using actual translation keys such as "ExerciseNotifications", "LectureNotifications", "InstructorExclusiveNotifications"?

Reply:
This is related to the way I designed the .defaut.ts settings
![image](https://user-images.githubusercontent.com/65814168/133658468-9f3d2517-4a40-4876-b540-ad2697d6a549.png)
I want to make this and future default settings to be easy to write and read (human-readable not lowerCamelCase)
Because these strings e.g. 'Exercise Notifications' or 'Get notified if a new exercise has been created or started' are easier to type and read than 'getNotifiedIfANewExerciseHasBeenCreatedOrStarted IMO, but if you and others (especially Stephan) tell me to change it to lowerCamelCase I will do so.

That was the entire reason why I played around with a (custom, then imported) new camelCase pipe to keep the default.ts readable and the translation JSON in camelcase format.
(here a picture of the template where the translation takes place) 
![image](https://user-images.githubusercontent.com/65814168/133659197-95849d92-d9ab-490e-8b4b-4b9692fc8709.png)